### DISCUSSION TITLE
changing domain fro

### DISCUSSION THREAD

**Discussion by jpicao** [EasyEngine/easyengine#1565] [Support Requests]
Hi im trying to change the domain on a wordpress site that we have on Digitalocean it was made using easy engine the issue was that my clients domain had expired and that they got a new one saying that they cant get the old domain anymore, I've tried to google it ajnd use the one on the old support forum but they don't work im quite  new to this so im at a loss on what to do please help, i have attached a txt file with the system info i got  thank you in advance 

System Information
[sysinfo.txt](https://github.com/EasyEngine/easyengine/files/6957717/sysinfo.txt)

**Comment by kirtangajjar** [comment]
@jpicao If you know both original domain name and IP of the site, You can access the old site by adding an entry in `/etc/hosts`.

**Reply by jpicao** [reply]
hello @kirtangajjar  thanx for replying, where do i add this entry?

**Reply by kirtangajjar** [reply]
Check this article to see how to add a host entry. It has section on Windows, Linux and MacOS, whichever OS you use - https://docs.rackspace.com/support/how-to/modify-your-hosts-file/

**Reply by jpicao** [reply]
hi we are using linux, i tried it irt dint work we actually have 2 sites in this one droplet, the domain keeps on being redirected to the other site, i tried to modify the host with the new domain it still the same thing.  is there any other options to change it?

**Reply by jpicao** [reply]
hi @kirtangajjar  any updates? thank you so much for the help by the way

**Reply by kirtangajjar** [reply]
@jpicao Redirection issues can have multiple reasons. You may need to debug it on your own from here :)

**Reply by kirtangajjar** [reply]
You added the host entry in your local system right? You don't need to add it on server. After adding host entry, try to open the site in Incognito window. Sometimes it's possible that the redirection might be caused by browser cache.

**Reply by kirtangajjar** [reply]
Best way to do it will be by curl:

```
curl -XGET -IL https://example.com
```

See if it gives you `200 OK` response

**Reply by jpicao** [reply]
hm i tried removing the other site temporarily and only have one site but it still the same it doent display the old content, i also tried adding a site with new domain and it works it dnt redirect, im quite cinfused now

**Reply by jpicao** [reply]
hi @kirtangajjar  is it posible to change this 
![gg](https://user-images.githubusercontent.com/49826794/130121762-be565e37-307a-4721-b8fc-1846e739cbc2.png)

**Comment by kirtangajjar** [comment]
No, it's not. You can:

1. Take a backup of the old site. You can use any method to take backups, a
plugin <https://wordpress.org/plugins/updraftplus/> might help.
2. Create a new site with new domain name
3. Import the backup in the new site.

**Comment by kirtangajjar** [comment]
P.S. You can use wp-cli to install, activate and use the above mentioned
backup plugin. So you can take backup of the server without accessing it's
UI, just using wp-cli

**Comment by jpicao** [comment]
hi @kirtangajjar  the problem is i cant access the site with old domain anymore, that domaion has expired and we had bhough a new one since its not available anymore

**Comment by jpicao** [comment]
hi @kirtangajjar  is it posile to just copy the wp site to a new domain?
